restcomm/restcomm.http/src/main/java/org/restcomm/connect/http/SupervisorEndpoint.java

```diff
@@ -184,7 +187,7 @@ protected Response registerForUpdates(final String accountSid, final Multivalued
     //Register a remote location where Restcomm will send monitoring updates for a specific Call
     protected Response registerForCallUpdates(final String accountSid, final String callSid, final MultivaluedMap<String, String> data, MediaType responseType) {
         secure(daos.getAccountsDao().getAccount(accountSid), "RestComm:Read:Calls");
```

Comment:
@otsakir code is updated, do you thing we are good to merge here now or we have anything else?
thanks
Maria